src/components/reusables/CompactDeletableListItem/CompactDeletableListItem.tsx

```diff
@@ -16,7 +16,7 @@ export enum ListTextColor {
 export interface CompactDeletableListItemProps {
   removeCallback?: (() => void) | null;
   leftText: string;
-  middleText: string;
+  middleText?: string | boolean;
```

Comment:
Why a boolean?
Does that mean I can feed the value `true` as a prop and expect a default middle text?

And if not, and this is only meant for the user of the component to be able to indicate they don't want any middle text, can't that just be achived by not passing a string at all? An undefined string is after all evaluated to false anyway.